mir-module/src/main/java/org/mycore/mir/migration/MIRMigrateStaticHistoryContent.java

```diff
@@ -0,0 +1,66 @@
+package org.mycore.mir.migration;
+
+import jakarta.persistence.EntityManager;
+import jakarta.persistence.criteria.CriteriaBuilder;
+import jakarta.persistence.criteria.CriteriaQuery;
+import jakarta.persistence.criteria.Predicate;
+import jakarta.persistence.criteria.Root;
+import jakarta.servlet.ServletContext;
+import org.mycore.backend.jpa.MCREntityManagerProvider;
+import org.mycore.common.events.MCRStartupHandler.AutoExecutable;
+import org.mycore.services.queuedjob.MCRJob;
+import org.mycore.services.queuedjob.MCRJobQueue;
+import org.mycore.services.queuedjob.MCRJobQueueManager;
+import org.mycore.services.queuedjob.MCRJobStatus;
+import org.mycore.services.queuedjob.MCRJob_;
+import org.mycore.util.concurrent.MCRTransactionableRunnable;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Class creates {@link MCRJob} in {@link MCRJobQueue} that migrates all static history content.
+ *
+ * @author shermann (Silvio Hermann)
+ * */
+public class MIRMigrateStaticHistoryContent implements AutoExecutable {
+
+    @Override
+    public String getName() {
+        return MIRMigrateStaticHistoryContent.class.getName();
+    }
+
+    @Override
+    public int getPriority() {
+        return 0;
```

Comment:
Should probably have a lower priority, since `org.mycore.services.queuedjob.MCRJobQueueInitializer` also uses priority `0` and this `AutoExecutable` relies on the job queue to be initialized.